docs/modules/password_auth_provider_callbacks.md

```diff
@@ -162,10 +162,38 @@ return `None`.
 If multiple modules implement this callback, they will be considered in order. If a
 callback returns `None`, Synapse falls through to the next one. The value of the first
 callback that does not return `None` will be used. If this happens, Synapse will not call
-any of the subsequent implementations of this callback. If every callback return `None`,
+any of the subsequent implementations of this callback. If every callback returns `None`,
 the username provided by the user is used, if any (otherwise one is automatically
 generated).
 
+### `get_display_name_for_registration`
```

Comment:
```suggestion
### `get_displayname_for_registration`
```

Although I'm not sure why this was changed? `display_name` seems clearer? Does the spec call it `displayname` or something?

Reply:
Yeah, the spec is consistently writing it `displayname` as far as I can tell.

Reply:
If only that were true: https://github.com/matrix-org/matrix-doc/issues/763

Reply:
:sob: